pkg/controller/lifecycle/actuator.go

```diff
@@ -57,26 +58,57 @@ type actuator struct {
 	config        *rest.Config
 	decoder       runtime.Decoder
 	serviceConfig config.Configuration
+	shootConfig   *v1alpha1.Configuration
 	oauth2secret  *config.OAuth2Secret
 	provider      filterListProvider
 	logger        logr.Logger
+	scheme        *runtime.Scheme
 }
 
 // Reconcile the Extension resource.
 func (a *actuator) Reconcile(ctx context.Context, _ logr.Logger, ex *extensionsv1alpha1.Extension) error {
 	blackholingEnabled := true
+	staticFilterList := []config.Filter{}
 	pspEnabled := true
 	secretData := map[string][]byte{
 		constants.KeyIPV4List: []byte("[]"),
 		constants.KeyIPV6List: []byte("[]"),
 	}
+
+	a.shootConfig = &v1alpha1.Configuration{}
```

Comment:
Why not just use a local variable?

```suggestion
	shootConfig := &v1alpha1.Configuration{}
```